magento/pwa-studio
# PWA-2355 UPWARD JS resolver for Computed type

## Description
To bring the JS implementation of the UPWARD spec closer to the PHP implementation, we've added the Computed resolver to it as well. The resolver simply returns an empty string so the application can continue without error.

## Related Issue







Closes PWA-2355

## Acceptance





### Verification Stakeholders


@zetlen

### Specification

Changes to `upward-spec` and/or `upward-js` packages must be reviewed

## Verification Steps






#### Test scenario(s) for direct fix/feature
- Run `DEBUG=upward-js:ComputedResolver yarn stage:venia`
- Visit front-end
- Should see `upward-js:ComputedResolver Computed resolver is meant for UPWARD PHP only` in the terminal
- Site should function normally, and inlined data should be missing from source (see screenshot for reference)
![image](https://user-images.githubusercontent.com/68294336/140093974-76f5cb5f-befe-41e5-9e6c-1578ca9b8d15.png)

#### Test scenario(s) for any existing impacted features/areas
- Test using UPWARD PHP
- Inlined data should be available (Can look for global variable `INLINED_PAGE_TYPE` in browser console)

## Checklist
